src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -150,6 +152,8 @@ public function getPriority(): int
 
     protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     {
+        $this->multilinedTokens = [];
+
         $expectedTokens = [T_LIST, T_FUNCTION, CT::T_USE_LAMBDA, T_FN, T_CLASS];
 
         for ($index = $tokens->count() - 1; $index > 0; --$index) {
```

Comment:
```suggestion
        $tokenCount = $tokens->count();
        for ($index = 1; $index < $tokenCount; ++$index) {
```
There is no need to introduce `multilinedTokens`, reversing this loop would be enough.